lib/core-server/src/utils/StoryIndexGenerator.ts

```diff
@@ -138,6 +147,24 @@ export class StoryIndexGenerator {
     // Pull out each file's stories into a list of stories, to be composed and sorted
     const storiesList = await this.ensureExtracted();
 
+    // Compute other imports
+    const allImports = storiesList.reduce((acc, entry) => {
+      Object.values(entry).forEach((story) => {
+        acc[scrubFileExtension(story.importPath)] = story.importPath;
+        return acc;
+      });
+      return acc;
+    }, {} as Record<string, Path>);
+
+    storiesList.forEach((entry) => {
+      Object.values(entry).forEach((story) => {
+        // eslint-disable-next-line no-param-reassign
+        story.storiesImports = story.storiesImports
```

Comment:
I feel like maybe we should store the "import paths without extension" in a different key so that we don't get confused when debugging why sometimes they have extensions and sometimes they don't? Maybe `story.importsWithoutExtension`?